client/components/AdoptionForm.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react"
+
+const AdoptionForm = () => {
+  const [adoptionData, setAdoptionData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    homeStatus: "own"
+  })
+  let errorContainer = null
+  const [errors, setErrors] = useState([])
+  if (errors.length !== 0) {
+    errorContainer = errors.map(error => {
+      return <p>{error}</p>
```

Comment:
I would look back at the React Form Validations with Hooks article for an example reusable ErrorList component